jobs/build_ova/prepare_ova_post_test.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash +xe
```

Comment:
Please add comments of the ENV Variable's meaning , as the **shell script usage**.
example: $External_vSwitch, $NODE_NAME